src/components/audio-feedback.js

```diff
@@ -28,23 +30,51 @@ export function getAudioFeedbackScale(fromObject, toObject, minScale, maxScale,
   return Math.min(maxScale, minScale + (maxScale - minScale) * volume * 8 * distance);
 }
 
+function updateVolume(component) {
+  const newVolume = calculateVolume(component.analyser, component.levels);
+
+  component.loudest = Math.max(component.loudest, newVolume);
+  component.volume = component.loudest === 0 ? 0 : THREE.Math.mapLinear(newVolume, 0, component.loudest, 0, 1);
+
+  const s = 0.3;
+  component.volume = s * component.volume + (1 - s) * component.prevVolume;
```

Comment:
Is this lerp a noticeable improvement, and is 0.3 a fine-tuned value?  (I know this code existed before this PR, just wondering if you messed with this at all)

Reply:
Did not mess with the value, but did notice it feels better to me with the lerp than without.